x-pack/elastic-agent/docs/tab-widgets/stop.asciidoc

```diff
@@ -41,6 +41,11 @@ Get the process ID (PID) of the `elastic-agent` process:
 ps | grep elastic-agent
 ----
 
+//REVIEWERS: This doesn't work on macOS. ^^ Does not seem to return the ID
+//of a actual process (ID doesn't match what I see in the activity monitor). 
+
+//TODO: move deb/rpm tabs to the end of the widget.
+
```

Comment:
I think it's because it's run by another user, did you try with sudo?

Reply:
Ohhhhh. I don't think I did. Maybe I will show that in the example.